src/main/features/core/tray.js

```diff
@@ -138,7 +142,9 @@ Emitter.on('audiooutput:list', (event, devices) => {
       });
     }
   });
-  setContextMenu();
+  if (appIcon) {
```

Comment:
I was just preserving structure, but the one you highlighted is actually inside a function, not at the top level so it will have to remain.  I'll condense the others.